### PR TITLE
venera: init at 1.0.8

### PR DESCRIPTION
**Pull request by ghost**
A comic reader that support reading local and network comics. https://github.com/venera-app/venera

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by lafrenierejm** [comment]
Are all three commits necessary for venera? It looks like you might have accidentally pushed some unrelated commits to this branch; de8305c5a1809a94b1a7544b75ef7a12085a6cd3, for example, only touches hiddify-app.

**Comment by Aleksanaa** [comment]
Could you try something like

<details>
<summary>patch</summary>


```patch
diff --git a/pkgs/development/compilers/dart/package-source-builders/sqlite3_flutter_libs/CMakeLists.txt b/pkgs/dev
elopment/compilers/dart/package-source-builders/sqlite3_flutter_libs/CMakeLists.txt
new file mode 100644
index 000000000000..4e8a182593f0
--- /dev/null
+++ b/pkgs/development/compilers/dart/package-source-builders/sqlite3_flutter_libs/CMakeLists.txt
@@ -0,0 +1,24 @@
+set(PROJECT_NAME "sqlite3_flutter_libs")
+project(${PROJECT_NAME} LANGUAGES C CXX)
+
+set(PLUGIN_NAME "sqlite3_flutter_libs_plugin")
+
+find_package(PkgConfig REQUIRED)
+
+pkg_check_modules(SQLITE REQUIRED sqlite3)
+
+add_library(SQLITE)
+
+add_library(${PLUGIN_NAME} SHARED
+  "sqlite3_flutter_libs_plugin.cc"
+)
+
+target_compile_features(${PLUGIN_NAME} PUBLIC cxx_std_14)
+target_compile_options(${PLUGIN_NAME} PRIVATE "$<$<NOT:$<CONFIG:Debug>>:-O3>")
+
+set_target_properties(${PLUGIN_NAME} PROPERTIES
+  CXX_VISIBILITY_PRESET hidden)
+target_compile_definitions(${PLUGIN_NAME} PRIVATE FLUTTER_PLUGIN_IMPL)
+target_include_directories(${PLUGIN_NAME} INTERFACE
+  "${CMAKE_CURRENT_SOURCE_DIR}/include")
+target_link_libraries(${PLUGIN_NAME} PRIVATE flutter)
diff --git a/pkgs/development/compilers/dart/package-source-builders/sqlite3_flutter_libs/default.nix b/pkgs/development/compilers/dart/package-source-builders/sqlite3_flutter_libs/default.nix
index faa5cb756a83..9fddae06d2ce 100644
--- a/pkgs/development/compilers/dart/package-source-builders/sqlite3_flutter_libs/default.nix
+++ b/pkgs/development/compilers/dart/package-source-builders/sqlite3_flutter_libs/default.nix
@@ -1,27 +1,14 @@
-{
-  stdenv,
-  lib,
-  fetchurl,
-}:
+{ stdenv }:
 
 { version, src, ... }:
 
-let
-  sqlite-autoconf = fetchurl {
-    url = "https://sqlite.org/2024/sqlite-autoconf-3470000.tar.gz";
-    hash = "sha256-g+shpvamSfUG34vTqrhaCPdVbO7V29jep0PqAD/DqVc=";
-  };
-in
 stdenv.mkDerivation {
   pname = "sqlite3_flutter_libs";
   inherit version src;
   inherit (src) passthru;
 
   postPatch = ''
-    substituteInPlace linux/CMakeLists.txt \
-      --replace-warn "https://sqlite.org/2024/sqlite-autoconf-3470000.tar.gz" "file://${sqlite-autoconf}" \
-      --replace-warn "https://sqlite.org/2024/sqlite-autoconf-3460000.tar.gz" "file://${sqlite-autoconf}" \
-      --replace-warn "https://www.sqlite.org/2022/sqlite-autoconf-3390000.tar.gz" "file://${sqlite-autoconf}"
+    cp ${./CMakeLists.txt} linux/CMakeLists.txt
   '';
 
   installPhase = ''
```

</details>

My network is poor today, cannot pull all dependencies, so no luck

File: pkgs/development/compilers/dart/package-source-builders/sqlite3_flutter_libs/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+set(PLUGIN_NAME "sqlite3_flutter_libs_plugin")
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(SQLITE REQUIRED sqlite3)
+
+add_library(${PLUGIN_NAME} SHARED
+  "sqlite3_flutter_libs_plugin.cc"
+)
+
+# Not applying the standard settings because we compile sqlite3
```
Comment (Aleksanaa) [inline pkgs/development/compilers/dart/package-source-builders/sqlite3_flutter_libs/CMakeLists.txt:16]:
```suggestion
```
We don't

File: pkgs/development/compilers/dart/package-source-builders/sqlite3_flutter_libs/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# which fails with -Wall -Werror. But we apply most of them:
```
Comment (Aleksanaa) [inline pkgs/development/compilers/dart/package-source-builders/sqlite3_flutter_libs/CMakeLists.txt:17]:
```suggestion
```

**Comment by Aleksanaa** [comment]
So does it work?

File: pkgs/by-name/ve/venera/package.nix [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      keywords = [
+        "Flutter"
+        "comic"
+        "images"
+      ];
+    })
+  ];
+
+  extraWrapProgramArgs = ''
+    --prefix LD_LIBRARY_PATH : "$out/app/${pname}/lib"
```
Comment (SuperSandro2000) [inline pkgs/by-name/ve/venera/package.nix:63]: We should not use pname here

**Pull request merged**